doc/user/sql/create-database.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "CREATE DATABASE"
+description: "`CREATE DATABASE` creates a new database namespace."
```

Comment:
I have a quibble with using the term "database namespace." A database namespace would be a schema—and indeed in Postgres the word "namespace" is overloaded to mean schema (see, e.g., https://www.postgresql.org/docs/12/catalog-pg-namespace.html). I think we want to just call these "databases", and rely on the conceptual framework below to explain what a database is. WDYT?